backend/src/main/scala/cromwell/backend/WorkflowBackendActor.scala

```diff
@@ -29,37 +31,68 @@ object WorkflowBackendActor {
   sealed trait AbortResponse extends WorkflowBackendActorResponse
   case class AbortSucceeded(jobKey: BackendJobDescriptorKey) extends AbortResponse
   case class AbortFailed(jobKey: BackendJobDescriptorKey, throwable: Throwable) extends AbortResponse
+
+  sealed trait InitializationResponse extends WorkflowBackendActorResponse
+  case object InitializationSuccess extends InitializationResponse
+  case class InitializationFailed(reason: Throwable) extends InitializationResponse
```

Comment:
It's coo', @cjllanwarne convinced me. I just wanted to make sure there was an actual use case in mind.